components/Wallet/EditWalletForm.tsx

```
@@ -87,60 +87,62 @@ export default function EditWalletForm ({ wallet, userAddresses, refreshWalletLi
+                      {userAddresses.map((addr, index) => (
+                        <div className={style.input_field} key={`edit-addr-${addr.id}`}>
+                          <input
+                            type='checkbox'
+                            value={addr.id}
+                            id={`addressIdList.${index}`}
+                            defaultChecked={thisWalletAddressIdList.includes(addr.id)}
+                            onChange={ (e) => handleSelectedAddressesChange(e.target.checked, addr.id) }
+                          />
+                          <label htmlFor={`addressIdList.${index}`}>
+                            {addr.paybuttons.map((conn) => (
+                              <div className={style.buttonpill}>{conn.paybutton.name}</div>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)